app/local/content/demos/ethregistry/ETHRegistryDemo.tsx

```diff
@@ -30,7 +30,7 @@ const Demo = () => {
         '0x225f137127d9067788314bc7fcc1f36746a3c3B5'
     );
     const [resolver, setResolver] = useState(
-        '0x225f137127d9067788314bc7fcc1f36746a3c3B5'
+        '0xce01f8eee7E479C928F8919abD53E553a36CeF67'
```

Comment:
```suggestion
        '0x231b0Ee14048e9dCcD1d247744d114a4EB5E8E63'
```

Reply:
Oh nice hadn't come across this, thanks!